ShellCaster.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\shell;
+
+use Symfony\Component\VarDumper\Caster\Caster;
+use yii\db\ActiveRecord;
+
+/**
+ * ShellCaster provides wrapper for casters of psysh
+ *
+ * @author Daniel Gomez Pan <pana_1990@hotmail.com>
+ * @since 2.0
+ */
+class ShellCaster
```

Comment:
Probably YiiCaster is more precise name since it works with Yii-specific objects.